src/test/java/hudson/plugins/emailext/ExtendedEmailPublisherTest.java

```diff
@@ -124,7 +124,7 @@ private void setUpSecurity() {
                                           // otherwise we would need to create users for each email address tested, to bypass SECURITY-372 fix:
                                           .grant(Jenkins.READ, Item.READ).everywhere().toAuthenticated()
                                           // TODO I had plans for tests where bob would approve scripts written by alice
-                                          .grant(Jenkins.ADMINISTER).everywhere().to("bob")
+                                          .grant(Jenkins.MANAGE).everywhere().to("bob")
```

Comment:
I think this tests should stay as they were (using `Jenkins.ADMINISTER`) which should still pass. And create a specific test for `Jenkins.MANAGE` (or an additional assert in an existing one).

Reply:
Agreed